src/entity.ts

```diff
@@ -376,11 +437,44 @@ export namespace entity {
     return value instanceof entity.Key;
   }
 
+  /**
+   * Convert a protobuf `integerValue`.
+   *
+   * @private
+   * @param {object} value The `integerValue` to convert.
+   */
+  function decodeIntegerValue(value: ValueProto) {
+    const num = Number(value.integerValue);
+    if (!Number.isSafeInteger(num)) {
+      throw new Error(
+        'We attempted to return all of the numeric values, but ' +
+          (value.propertyName ? value.propertyName + ' ' : '') +
+          'value ' +
+          value.integerValue +
+          ' is out of bounds of Number safe integer.\n' +
```

Comment:
Could you replace "Number safe integer" with "Number.MAX_SAFE_INTEGER"? It will give people something to Google if they're unfamiliar.

Reply:
Done